script/ci

```diff
@@ -2,8 +2,24 @@
 set -euo pipefail
 IFS=$'\n\t'
 
+prepare_build() {
+  if [ -d coverage ]; then
+    rm -rf coverage
+  fi
+}
+
+print_ruby_version() {
+  echo "Using $(ruby -v)"
+  echo
+}
+
+run_code_quality_checks() {
+  # bundle exec rubocop .
```

Comment:
is this ok?

Reply:
From what I can see from other projects like `router` or `controller` they have the same code.
Maybe the code quality is run with another tool?

Reply:
We don't lint all the Hanami gems with `rubocop` yet, as their 1.x version. At 2.x will be all linted.